src/Compilers/VisualBasic/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.vb

```diff
@@ -1535,5 +1535,31 @@ Block[B2] - Exit
                 ControlFlowGraphVerifier.VerifyGraph(compilation, expectedFlowGraph, actualFlowGraph)
             Next
         End Sub
+
+        <Fact, WorkItem(30309, "https://github.com/dotnet/roslyn/issues/30309")>
+        Public Sub TestSymbolStartAnalyzer_MyApplication()
+
+            Dim sources = <compilation>
+                              <file name="a.vb"><![CDATA[
+Namespace My
```

Comment:
@AlekseyTs Thanks for pointing to the VB MyTemplate code file. That helped me write this test correctly.